packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -129,6 +179,12 @@ To enable grouped batching:
     }
 ```
 
+## Note about performance and latency
+
+In terms of performance and impact on latency, Fluid does not make any guarantees or provide benchmarks as the effects of compression/chunking/grouped batching are coupled with the nature of the payloads and the configured `FlushMode`. Therefore, customers must perform the required measurements and adjust the settings according to their scenarios.
+
+In general, compression offers a trade-off between higher compute costs, lower bandwidth consumption and lower storage requirements, while chunking slightly increases latency due to the overhead of splitting an op, sending the chunks and reconstructing them on each client. Grouped batching heavily decreases the number of ops observed by the server (therefore decreasing the odds of throttling) and slightly decreases the bandwidth requirements as it merges all the ops in a batch into a single op and also eliminates the op envelope overhead.
```

Comment:
I think chunking should not be optional - it's correctness feature (all correctness features should not be optional).
We should only expose knobs (at final step) to when to engage compression (starting size), and that's probably it? When compression is disabled, we should still provide all the same promises of correctness (any payload sizes) and ordering, grouping all the ops (using "trivial" compression) and eventually getting rid from "batching" element of protocol (i.e., simplify protocol and for service does not care about sequencing all ops in one "submitOp" message together - the simpler protocol the better). But this is future (2-3 years for FRS)